src/hotspot/share/opto/graphKit.cpp

```diff
@@ -2350,9 +2350,9 @@ void GraphKit::round_double_arguments(ciMethod* dest_method) {
       const Type *targ = tf->domain()->field_at(j + TypeFunc::Parms);
       if (targ->basic_type() == T_DOUBLE) {
         // If any parameters are doubles, they must be rounded before
-        // the call, dstore_rounding does gvn.transform
+        // the call, precision_rounding does gvn.transform
         Node *arg = argument(j);
-        arg = dstore_rounding(arg);
+        arg = precision_rounding(arg);
```

Comment:
Hold on. This should be `dprecision_rounding`, with a `d`?

Reply:
Good catch!

Reply:
> Hold on. This should be `dprecision_rounding`, with a `d`?

Ah, yes.

Thanks @iwanowww and @shipilev for your review.
All the comments had been addressed.